bpf/ut/nat_test.go

```diff
@@ -801,6 +805,15 @@ func TestNATNodePort(t *testing.T) {
 			pktR := gopacket.NewPacket(res.dataOut, layers.LayerTypeEthernet, gopacket.Default)
 			fmt.Printf("pktR = %+v\n", pktR)
 
+			ethL := pktR.Layer(layers.LayerTypeEthernet)
+			Expect(ethL).NotTo(BeNil())
+			ethR := ethL.(*layers.Ethernet)
+			Expect(ethR).To(layersMatchFields(&layers.Ethernet{
+				SrcMAC:       macUntouched, // Source is set byt net stack and should not be touched.
+				DstMAC:       macScr,
```

Comment:
Should this be `s/macScr/macSrc/g`? If not, what does "scr" mean here?

Reply:
yup typos,thanks